api/SubtleCrypto.json

```diff
@@ -1029,6 +1043,24 @@
             "deprecated": false
           }
         }
+      },
+      "worker_support": {
+        "__compat": {
+          "description": "Available in workers",
```

Comment:
@Elchi3 , is there any standard (or conventional) way to represent or describe this condition? I think this is fine as it is, but if there is a standard or convention here we ought to follow it.

Reply:
This was based on `schemas/compat-data-schema.md` [here](https://github.com/mdn/browser-compat-data/blob/master/schemas/compat-data-schema.md#api-specific-subfeatures) and [here](https://github.com/mdn/browser-compat-data/blob/master/schemas/compat-data-schema.md#version_added). Is that what you mean?

Reply:
Yes, that's exactly what I meant :).